src/Domain/Donation.php

```diff
@@ -13,11 +14,41 @@
 class Donation {
 	use FreezableValueObject;
 
+	// status for direct debit
+	const STATUS_NEW = 'N';
+
+	// status for bank transfer
+	const STATUS_PROMISE = 'Z';
+
+	// statuses for external payments
+	const STATUS_EXTERNAL_INCOMPLETE = 'X';
+	const STATUS_EXTERNAL_BOOKED = 'B';
+
+	const STATUS_MODERATION = 'P';
+	const STATUS_DELETED = 'D';
+
+	private $status;
 	private $amount;
 	private $interval = 0;
 	private $paymentType;
 	private $personalInfo;
+	private $optIn;
 	private $bankData;
+	private $tracking;
+	private $source;
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $color;
+	private $skin;
+	private $layout;
```

Comment:
Would this all fit in a TrackingData object? It is all tracking stuff right, or is some of it not?